internal/capability/operator_install.go

```diff
@@ -37,7 +37,7 @@ func OperatorInstallAllFromCatalog(catalogSource string, catalogSourceNamespace
 		// of it's operator dedicated goroutine
 		err := OperatorInstall(subscription, c)
 		if err != nil {
-			logger.Errorf("Package %s, channel %s, install mode %s - FAILED to complete test", subscription.Package, subscription.Channel, subscription.InstallModeType)
+			logger.Errorw("Package", subscription.Package, "channel", subscription.Channel, "install mode", subscription.InstallModeType)
```

Comment:
Errorw signature is (msg string, keysAndValues ...interface{}). A message is needed and the key/value pairs should be lowercase with no spaces; package vs Package and installmode vs install mode